src/app/collective-rewards/shared/actions.ts

```diff
@@ -52,16 +53,10 @@ const apolloQuery = apolloGQL`
     }
   }
 `
-
-const fetchCrTheGraphEndpoint = `${process.env.THE_GRAPH_URL}/${process.env.THE_GRAPH_API_KEY}/${process.env.THE_GRAPH_ID}`
 const client = makeClient(fetchCrTheGraphEndpoint)
```

Comment:
Do we need to create a client per request? Right now, I don't see it as an issue, but maybe we can optimize this.

Reply:
Good point, I'm going to use the same client across the dapp 👍🏼